manager/allocator/network.go

```diff
@@ -574,6 +575,11 @@ func (a *Allocator) allocateNetworks(ctx context.Context, existingOnly bool) err
 			log.G(ctx).WithField("existingOnly", existingOnly).WithError(err).Errorf("failed allocating network %s during init", n.ID)
 			continue
 		}
+
+		// If we're restoring, we don't need to commit this to the store again.
+		if existingOnly {
```

Comment:
Totally non-blocking nit - would it make it less verbose/have less loop redirection if we inverted to do:

```golang
if !existingOnly {
    allocatedNetworks = append(allocatedNetworks, n)
}
```